examples/gitlab-centos7-atomicapp/Nulecule

```diff
@@ -0,0 +1,63 @@
+---
+specversion: 0.0.2
+id: gitlab-atomicapp
+
+metadata:
+  name: Gitlab App
+  appversion: 1.2.0
+  description: Gitlab.
+graph:
+  - name: redis
```

Comment:
There is a redis Atomic App, please reuse: https://github.com/navidshaikh/nulecule/blob/fix-160/examples/redis-centos7-atomicapp/Nulecule